lib/index.js

```diff
@@ -355,18 +382,21 @@ Publisher.prototype.publish = function (headers, options) {
 
 /**
  * Sync file in stream with file in the s3 bucket
- * @param {String} prefix
+ * @param {Object} options
+ * @param {String} options.prefix prefix to sync a specific directory
+ * @param {Array} options.whitelist list of expressions that match against files that should not be deleted
```

Comment:
`whitelist` may not sound too representative... but couldn't think of a better name for the param.